apps/client/ui/Header/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import { LogoIcon } from '@/ui/Icons/Logo/Logo'
+import { NavMenu } from '@/ui/NavigationMenu/NavMenu'
+import { InputSearch } from '@/ui/InputSearch/InputSearch'
+import { Avatar } from '@/ui/Avatar/Avatar'
+import { useState } from 'react'
+import type { FC } from 'react'
+
+export const Header: FC = () => {
+  const [activeMenuItem, setActiveMenuItem] = useState<string | null>('Movies')
+  const [isDropdownOpen, setIsDropdownOpen] = useState<boolean>(false)
+
+  const handleMenuHover = (item: string) => {
+    setActiveMenuItem(item !== activeMenuItem ? item : null)
```

Comment:
better use callback argument with previous value here instead of using `activeMenuItem` value 👌 to be honest in this case it has no difference but it's better practice to always upadte state this way to have the habit in case it will help you prevent bugs with batching 😄